src/Analyzers/CSharp/Tests/UseConditionalExpression/UseConditionalExpressionForReturnTests.cs

```diff
@@ -502,6 +502,146 @@ int M()
+                """
+                class C
+                {
+                    int M()
+                    {
+                        int x = 0;
+                        int y = 0;
+                        return checked(x == y) ? 0 : 1;
+                    }
+                }
+                """, parseOptions: CSharp8);
+        }
+
+        [Fact, WorkItem("https://github.com/dotnet/roslyn/issues/70748")]
+        public async Task TestMissingWithUncheckedInIf()
+        {
+            await TestInRegularAndScriptAsync(
+                """
+                class C
+                {
+                    int M()
+                    {
+                        int x = 0;
+                        int y = 0;
+                        [||]if (unchecked(x == y))
+                        {
+                            return 0;
+                        }
+                        else
+                        {
+                            return 1;
+                        }
+                    }
+                }
+                """,
+                """
+                class C
+                {
+                    int M()
+                    {
+                        int x = 0;
+                        int y = 0;
+                        return unchecked(x == y) ? 0 : 1;
+                    }
+                }
+                """, parseOptions: CSharp8);
+        }
+
+        [Fact, WorkItem("https://github.com/dotnet/roslyn/issues/70748")]
+        public async Task TestMissingWithCheckedInTrueStatement()
+        {
+            await TestInRegularAndScriptAsync(
+                """
+                class C
+                {
+                    int M()
+                    {
+                        int x = 0;
+                        int y = 0;
+                        [||]if (x == y)
+                        {
+                            return checked(x - y);
+                        }
+                        else
+                        {
+                            return 1;
+                        }
+                    }
+                }
+                """,
+                """
+                class C
+                {
+                    int M()
+                    {
+                        int x = 0;
+                        int y = 0;
+                        return x == y ? checked(x - y) : 1;
+                    }
+                }
+                """, parseOptions: CSharp8);
+        }
+
+        [Fact, WorkItem("https://github.com/dotnet/roslyn/issues/70748")]
+        public async Task TestMissingWithUncheckedInTrueStatement()
+        {
+            await TestInRegularAndScriptAsync(
+                """
+                class C
+                {
+                    int M()
+                    {
+                        int x = 0;
+                        int y = 0;
+                        [||]if (x == y)
+                        {
+                            return unchecked(x - y);
```

Comment:
i don't see a test where teh checked/unchecked is in the false statement.  can you add?

Reply:
done